### DISCUSSION TITLE
Allow the ability to make a music track override others.

### DISCUSSION THREAD

**Discussion by Jasii-S** [FakeFishGames/Barotrauma#9980] [Ideas and suggestions]
Adding a tag or a value (similar to sprite depth) for music would be super helpful for mods that edit music.
 It allows creators to be more creative. For example, all music stops when flooded and instead plays the flooded music. Or when at an abandoned outpost all music stops playing and the abandoned outpost music only plays. As it is now it is not an issue for vanilla because vanilla has music only played in calm situations, otherwise, horror tunes play which do not really overlap the music. For mods looking to spice things up a bit and add combat music, overlapping music could be a huge issue and can cause creators to be forced to be simple and uninventive.

**Comment by Udrakan** [comment]
Generally having the ability to influence which music plays would be awesome for story building and I think this should also be doable via events. Story/atmosphere mods will be beyond amazing if we get control over this.

**Reply by Jasii-S** [reply]
Im having an issue with my mod where I have to remove tracks or not have any music at all for some because the music keeps overlapping. Gets real annoying real fast. Having more ability to modify stuff like that would be sososo helpful.
